webapp/src/shared/components/Row/Row.tsx

```diff
@@ -0,0 +1,55 @@
+import Icon, { IconVariant, IconSize } from '../Icon/Icon';
+import './Row.css';
+import { TextColor, TextVariant, TextWeight } from '../Text/Text';
+import { default as Text } from '../Text/Text';
+import { SmallAvatar } from '../Avatar/Avatar';
+import { StatusVariant } from '../Status/Status';
+
+type RowProps = {
+  iconVariant?: IconVariant;
+  onClick?: () => void;
+  avatarUrl?: string;
+  avatarStatus?: StatusVariant;
+  title?: string;
+  subtitle?: string;
+};
+
+export default function Row({
+  iconVariant,
+  onClick,
+  avatarUrl,
+  avatarStatus,
```

Comment:
If we finally go with this option, I'd group related props at least. This is -> `AvatarProps`